### DISCUSSION TITLE
I can't add a Domain to advanced configuration.

### DISCUSSION THREAD

**Discussion by miguel-resource** [vercel/community#7091] [Help]
### Page to Investigate

https://vercel.com/account/domains

### Steps to Reproduce

I'm trying to add a Domain in this link to can use "Advanced Configuration", but when I input the info and clicked "Add" I was redirect to this exactly URL https://vercel.com/[teamSlug]/[project]/settings/domains, but when I went to my project saw the basic Domain, but when I went to my project saw the basic Domain, but I only want use the "Advanced Configuration" .

![Captura de pantalla 2024-06-04 a la(s) 1 44 57 p m](https://github.com/vercel/community/assets/61020779/b4da22c5-f8c2-46a6-8f33-8dfbc1862acf)

**Comment by amyegan** [comment]
Hey @miguel-resource. I'm not sure I know what "Advanced Configuration" you're talking about. But you can add a custom domain by following the steps outlined here: [vercel.com/docs/projects/domains/add-a-domain](https://vercel.com/docs/projects/domains/add-a-domain)

**Reply by miguel-resource** [reply]
Hi @amyegan, but I only try to create a custom domain and add a MX DNS email, where I can do this?
